crates/zizmor/Cargo.toml

```diff
@@ -53,7 +53,7 @@ jsonschema.workspace = true
 line-index.workspace = true
 owo-colors.workspace = true
 regex.workspace = true
-reqwest = { workspace = true, features = ["blocking", "json", "rustls-tls"] }
+reqwest = { workspace = true, features = ["blocking", "json", "rustls-tls", "rustls-tls-native-roots"] }
```

Comment:
This seems fine, but noting: we'll probably want to use `rustls-platform-verifier` once available in reqwest.

Ref: https://github.com/seanmonstar/reqwest/issues/2159